src/plugins/teleop/Teleop.qml

```diff
@@ -0,0 +1,399 @@
+        if(backwardButton.checked)
+          backwardButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      ToolTip.visible: hovered
+      ToolTip.text: "Forward"
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: forwardButton.text
+      }
+    }
+    Button {
+      id: leftButton
+      text: "\u25C0"
+      checkable: true
+      Layout.row: 1
+      Layout.column: 0
+      onClicked: {
+        Teleop.angularDir = leftButton.checked ? 1 : 0
+        if(rightButton.checked)
+          rightButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: leftButton.text
+      }
+    }
+    Button {
+      id: rightButton
+      text: "\u25B6"
+      checkable: true
+      Layout.row: 1
+      Layout.column: 2
+      onClicked: {
+        Teleop.angularDir = rightButton.checked ? -1 : 0
+        if(leftButton.checked)
+          leftButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: rightButton.text
+      }
+    }
+    Button {
+      id: backwardButton
+      text: "\u25BC"
+      checkable: true
+      Layout.row: 2
+      Layout.column: 1
+      onClicked: {
+        Teleop.linearDir = backwardButton.checked ? -1 : 0
+        if(forwardButton.checked)
+          forwardButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: backwardButton.text
+      }
+    }
+    Button {
+      id: stopButton
+      text: "Stop"
+      checkable: false
+      Layout.row: 1
+      Layout.column: 1
+      onClicked: {
+        Teleop.linearDir = 0
+        Teleop.angularDir = 0
+        forwardButton.checked = false
+        leftButton.checked = false
+        rightButton.checked = false
+        backwardButton.checked = false
+        linearVelSlider.value = 0
+        angularVelSlider.value = 0
+        slidersSwitch.checked = false
```

Comment:
How about doing this for the arrows too? Right now we can get in a mixed situation if arrows are pressed while the sliders are enabled.

Reply:
Addressed